### ISSUE TITLE
Question About How to Generate Sheet Scaffolds

### ISSUE THREAD

**Issue by chenlindu** [Alistairj43/TPMS-Designer#10]
Hi Alistair, 

Thank you very much for your great work! 

I am trying to use your toolkit to gnerate a sheet Gyroid scaffolds for further study. However, I did not find a possible option for the sheet scaffold. I am really a new beginner of this filed, therefore, I really appreciate if you could offer me some help. 

Thank you so much in advance for your time and look forward for your reply.

Sincererly,
Chenlin

**Comment by Alistairj43** [comment]
Hi Chenlin,

I have used the terminology "surface" for the thin-walled type structure). Otherwise I'm not sure what you are asking - Can you share an example of what you mean by a "sheet Gyroid scaffold"?/what you are trying to do with them?

**Comment by chenlindu** [comment]
Hi Alistair,

Thank you so much for your reply!

I think I have solved the issue by reading the source code as I found the formula used for surface was identical to the sheet mentioned in this [paper](https://doi.org/10.1016/j.jmbbm.2021.104871). The formula is as follows:

```math
\phi_{sheet}(x,y,z)=(\phi_{TPMS}(x,y,z)+r(x,y,,z))\times(\phi_{TPMS}(x,y,z)-r(x,y,,z))>0
```


Once again, I would like to express my gratitude for your assistance and great work!

Sincererly,
Chenlin
